### PR TITLE
:sparkles: issue #59 provider support to add start duration in the da…

### LINKED ISSUE #57: Change default start hour in DayView
Is it possible to change the default start hour in DayView from 1am to something like 8am? I looked into the TimeLineMark, but am not what I should update it to. 
Screen Shot 2022-04-01 at 12 28 59 PM

### PR DESCRIPTION
**Pull request by DhavalRKansara**
…y view calendar.

# Description

## Checklist

- [x] The title of my PR starts with a [Conventional Commit] prefix (`fix:`, `feat:`, `docs:` etc).
- [x] I have followed the [Contributor Guide] when preparing my PR.
- [x] I have updated/added tests for ALL new/updated/fixed functionality.
- [x] I have updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
- [x] I have updated/added relevant examples in `examples` or `docs`.

## Breaking Change?

- [ ] Yes, this PR is a breaking change.
- [x] No, this PR is not a breaking change.

## Related Issues

Closes #57

[Contributor Guide]: https://github.com/SimformSolutionsPvtLtd/flutter_calendar_view/blob/master/CONTRIBUTING.md
[Conventional Commit]: https://conventionalcommits.org

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,77 +129,83 @@ visit [Calendar View Example](https://simformsolutionspvtltd.github.io/flutter_c
...
-    onPageChange: (date, pageIndex) => print("$date, $pageIndex"),
-    onCellTap: (events, date) {
-        // Implement callback when user taps on a cell.
-        print(events);
-    },
-    startDay: WeekDays.sunday, // To change the first day of the week.
-    // This callback will only work if cellBuilder is null.
-    onEventTap: (event, date) => print(event),
-    onDateLongPress: (date) => print(date),
+MonthView
```
Comment (PRBaraiya) [inline README.md:132]: undo this.

File: README.md [hunk 2]
```diff
@@ -125,77 +129,83 @@ visit [Calendar View Example](https://simformsolutionspvtltd.github.io/flutter_c
...
-    showVerticalLine: true, // To display live time line in day view.
-    showLiveTimeLineInAllDays: true, // To display live time line in all pages in day view.
-    minDay: DateTime(1990),
-    maxDay: DateTime(2050),
-    initialDay: DateTime(2021),
-    heightPerMinute: 1, // height occupied by 1 minute time span.
-    eventArranger: SideEventArranger(), // To define how simultaneous events will be arranged.
-    onEventTap: (events, date) => print(events),
-    onDateLongPress: (date) => print(date),
+DayView
```
Comment (PRBaraiya) [inline README.md:159]: ^^

File: README.md [hunk 3]
```diff
@@ -125,77 +129,83 @@ visit [Calendar View Example](https://simformsolutionspvtltd.github.io/flutter_c
...
-    width: 400, // width of week view.
-    minDay: DateTime(1990),
-    maxDay: DateTime(2050),
-    initialDay: DateTime(2021),
-    heightPerMinute: 1, // height occupied by 1 minute time span.
-    eventArranger: SideEventArranger(), // To define how simultaneous events will be arranged.
-    onEventTap: (events, date) => print(events),
-    onDateLongPress: (date) => print(date),
-    startDay: WeekDays.sunday, // To change the first day of the week.
+WeekView
```
Comment (PRBaraiya) [inline README.md:188]: ^^

File: README.md [hunk 4]
```diff
@@ -205,86 +215,97 @@ visit [documentation](https://pub.dev/documentation/calendar_view/latest/calenda
...
 ```dart
-WeekView(
-  weekDays: [
-    WeekDays.monday,
-    WeekDays.tuesday,
-    WeekDays.wednesday,
-    WeekDays.thursday,
-    WeekDays.friday,
-  ],
+WeekView
```
Comment (PRBaraiya) [inline README.md:298]: ^^

File: lib/src/day_view/day_view.dart [hunk 5]
```diff
@@ -776,6 +785,26 @@ class DayViewState<T extends Object?> extends State<DayView<T>> {
     );
   }
 
+  /// Animate to specific offset in a day view using the start duration
+  Future<void> startWithDuration(
```
Comment (PRBaraiya) [inline lib/src/day_view/day_view.dart:789]: can we change function name to animateToDuration

**Review by PRBaraiya — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (DhavalRKansara) [reply README.md:292]: Updated

File: README.md (hunk 3 above)
Comment (DhavalRKansara) [reply README.md]: Updated

File: README.md (hunk 2 above)
Comment (DhavalRKansara) [reply README.md]: Updated

File: README.md (hunk 1 above)
Comment (DhavalRKansara) [reply README.md]: Updated

**Review by PRBaraiya — APPROVED** [review]
LGTM!

**Pull request merged**